### PR TITLE
Reopen pull request (update domains) 

### PR DESCRIPTION
**Pull request by max21910**
Hi, I’m sorry I accidentally deleted my repository that so delete my pull request. Fortunately I had a backup here the file edit : 

`Update remove domains`

### CODE REVIEW

**Review by userbradley — CHANGES_REQUESTED** [review]
Thanks again! Are you able to remove the `.DS_store` file?

I think the below should fix that

```
git rm .DS_Store
git add *
git commit -m 'Removed the file mac loves to add'
git push
```

File: .DS_Store
[the remark this answers is not in the document]
Comment (max21910) [reply .DS_Store:1]: should be fix now :)

**Comment by userbradley** [comment]
image

LMAO - [This file only serve to get in the way](https://www-bleepingcomputer-com.translate.goog/news/security/google-drive-flags-macos-ds-store-files-for-copyright-violation/?_x_tr_sl=auto&_x_tr_tl=fr&_x_tr_hl=en-US&_x_tr_pto=wapp)

**Review by userbradley — APPROVED** [review]
Approved - Merging

**Pull request merged**